README.md

```diff
@@ -199,38 +199,29 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    (venv) $ ilab
    Usage: ilab [OPTIONS] COMMAND [ARGS]...
 
-   CLI for interacting with InstructLab.
+     CLI for interacting with InstructLab.
 
-   If this is your first time running InstructLab, it's best to start with `ilab config init` to create the environment.
+     If this is your first time running ilab, it's best to start with `ilab config init` to create the environment.
 
    Options:
-   --config PATH  Path to a configuration file.  [default:
-                  /home/user/.config/instructlab/config.yaml]
-   -v, --verbose  Enable debug logging (repeat for even more verbosity)
-   --version      Show the version and exit.
-   --help         Show this message and exit.
+     --config PATH  Path to a configuration file.  [default:
+                    /home/jhradilek/.config/instructlab/config.yaml]
```

Comment:
Best to leave this as the generic `user` and not your personal user account name.